daktari/check_runner.py

```diff
@@ -35,13 +35,20 @@ def try_run_check(self, check: Check):
 
     def run_check(self, check: Check):
         logging.info(f"Running check {check.name}")
-        result = check.check()
+        result = self.run_check_in_try(check)
         print_check_result(result)
         if result.status in (CheckStatus.PASS, CheckStatus.PASS_WITH_WARNING):
             self.checks_passed.add(check.name)
         else:
             self.all_passed = False
 
+    def run_check_in_try(self, check: Check) -> CheckResult:
+        try:
+            return check.check()
+        except Exception as err:
+            logging.debug("Exception running check", exc_info=True)
```

Comment:
Would it be helpful to include the check name in the debug message? I guess it'll be in an earlier log, but might make it super easy to spot where it's going wrong?